packages/manager/apps/dedicated/client/app/dedicated/server/install/image/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dedicated_server_install_image_title": "Installer une image personnalisée",
+  "dedicated_server_install_image_explain": "La technologie Bring Your Own Image (BYOI) vous permet de déployer des images cloud-ready directement sur vos serveurs dédiés. Vous pouvez ainsi utiliser votre solution bare metal comme ressource pour ces installations.",
+  "dedicated_server_install_image_doc": "Pour plus d’informations, <a href=\"{{ href }}\">cliquez ici pour consulter notre documentation</a>.",
+  "dedicated_server_install_image_submit": "Installer le système",
+  "dedicated_server_install_image_launch_install_error": "Une erreur est survenue lors du lancement de l'installation à partir d'une image personnalisée. <i>{{ message }}</i>",
```

Comment:
You can extract the `<i>{{ message }}</i>` from the translation, and use it directly in the template